micrometer-core/src/test/java/io/micrometer/core/instrument/binder/cache/EhCache2MetricsTest.java

```diff
@@ -0,0 +1,201 @@
+/**
+ * Copyright 2018 Pivotal Software, Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micrometer.core.instrument.binder.cache;
+
+import io.micrometer.core.instrument.FunctionCounter;
+import io.micrometer.core.instrument.Gauge;
+import io.micrometer.core.instrument.Tags;
+import io.micrometer.core.instrument.simple.SimpleMeterRegistry;
+
+import java.util.Random;
+
+import net.sf.ehcache.Cache;
+import net.sf.ehcache.CacheManager;
+import net.sf.ehcache.statistics.StatisticsGateway;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.springframework.test.util.ReflectionTestUtils;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+/**
+ * Tests for {@link EhCache2Metrics}.
+ *
+ * @author Oleksii Bondar
+ */
+class EhCache2MetricsTest extends AbstractCacheMetricsTest {
+
+    private static CacheManager cacheManager;
+    private static Cache cache;
+
+    private Tags expectedTag = Tags.of("app", "test");
+    private EhCache2Metrics metrics = new EhCache2Metrics(cache, expectedTag);
+
+    @Test
+    void reportMetrics() {
+        SimpleMeterRegistry registry = new SimpleMeterRegistry();
```

Comment:
Left-hand side of assignment can be `MeterRegistry`

Reply:
Thanks, addressed across all *CacheMetricsTest classes